tests/e2e/README.md

```diff
@@ -0,0 +1,93 @@
+# End-2-End test suite for InstantSearch
+
+This repository contains the end-2-end test suite for InstantSearch.
+
+## Installation
+
+The workspace automatically links the dependencies when you run `yarn` at the root of the repository.
+
+## Running the test suite
+
+In addition to the test suite this package contains two [WebdriverIO](https://webdriver.io) configurations:
+
+- `local`: Run the test suite on [Chrome browser](https://google.com/chrome) on your local machine.
+- `saucelabs`: Run the test suite on multiple browsers on the [Sauce Labs service](https://saucelabs.com).
+
+All configurations will run a static web server on your machine on port `5000` to serve the content of the `website` directory in your project, containing the InstantSearch demos to run our tests against.
```

Comment:
This is still worded as if this was a reusable package to install in your repo.

Reply:
It's still how we use the tests in this repo :)

Reply:
Not exactly because you no longer install it per repo. It's installed in the monorepo and will work with every new flavor that we migrate, without changes to any configuration (aside from adding new flavors to `flavors.js`) no?

I guess what I'm saying is that there's nothing more to install or set up for end-to-end tests to work in this repo. I'm not sure whether this information is still useful.